test/Microsoft.ML.Tests/Scenarios/Api/Estimators/DeserializationTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Calibrators;
+using Microsoft.ML.Data;
+using Microsoft.ML.RunTests;
+using Microsoft.ML.Trainers.FastTree;
+using Xunit;
+
+namespace Microsoft.ML.Tests.Scenarios.Api
+{
```

Comment:
Should these go in Microsoft.ML.Functional.Tests?
They seem to be in the same spirit of Rogan's tests.